man/common

```diff
@@ -0,0 +1,63 @@
+.It Fl e Ar expression
+Evaluates the specified expression.
+.It Fl r Ar file
+Requires the specified file. In other words, the file is checked for existence at the start of
+.Nm .
+.It Fl S Ar script
+Runs the specified script.
+.It Fl pa Ar directory
+Adds the specified directory to the beginning of the code path. If the directory already exists, it will be removed from its old position and put to the beginning.
+.Pp
+See also the function
+.Sy Code.prepend_path/1 .
+.It Fl pr Ar file
+Does the same thing as
+.Fl r
+.Pq see above
+but in parallel.
+.It Fl pz Ar directory
+Adds the specified directory to the end of the code path. If the directory already exists, it won't be neither removed from its old position nor put to the end.
```

Comment:
The main problem here is double negative. I think we have to get rid off the first one. Thus, the sentence should look like the following: "...it will be neither removed from its old position nor put to the end.".